_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -36,7 +36,12 @@ htmlHintIgnore:
 
 ## Applicability
 
-The rule applies to HTML `img` elements and HTML elements with the [semantic role][] of `img`, except for elements that are not [included in the accessibility tree][].
+The rule applies to HTML `img` elements and HTML elements with the [semantic role][] of `img` which render an image and are [included in the accessibility tree][].
```

Comment:
```suggestion
The rule applies to HTML `img` elements and HTML elements with the [semantic role][] of `img` which render an image and which are [included in the accessibility tree][].
```

Reply:
No problem - I'll give everyone a chance to comment then push an update